src/hooks/useVaultInformation.ts

```diff
@@ -16,28 +16,23 @@ interface VaultInformation {
 export function useVaultInformation(vault: VaultDTO): VaultInformation {
   const { user, vaults } = useContext(StoreContext);
   const { showAPR } = vaults.vaultsFilters;
-  const depositBalance = user.getBalance(vault.vaultToken);
-  let vaultBoost = showAPR ? vault.apr : vault.apy;
+  const { vaultToken, apr, minApr, apy, minApy, yieldProjection } = vault;
+  const { nonHarvestApr, nonHarvestApy, harvestPeriodApr, harvestPeriodApy } = yieldProjection;
 
-  let projectedBaseApr = 0;
-  if (vault.version === VaultVersion.v1_5) {
-    projectedBaseApr = vault.yieldProjection.harvestTokensPerPeriod
-      .filter((s) => !isBadgerSource(s))
-      .reduce((total, s) => (total += s.apr), 0);
-  }
+  const depositBalance = user.getBalance(vaultToken);
+  const boostContribution = getBoostContribution(vault, user.accountDetails?.boost ?? 0);
 
-  let projectedVaultBoost = projectedBaseApr > 0 ? projectedBaseApr : null;
-  if (user.accountDetails?.boost) {
-    vaultBoost = getUserVaultBoost(vault, user.accountDetails.boost, showAPR);
-    if (projectedVaultBoost) {
-      projectedVaultBoost += getProjectedVaultBoost(vault, user.accountDetails.boost);
-    }
-  }
+  const sourceApr = minApr ?? apr;
+  const sourceApy = minApy ?? apy;
+  const baseYield = showAPR ? sourceApr : sourceApy;
+  const vaultBoost = baseYield + boostContribution;
```

Comment:
boost same for both apr/apy?

Reply:
yes because its not effected by any kind of compounding frequency